recipes/aws-sdk-cpp/all/test_package/conanfile.py

```diff
@@ -1,17 +1,26 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake
```

Comment:
"CMakeDeps", "CMakeToolchain"?

Reply:
@mpimenov it's not needed this case, because they are declared in generators attribute already. As we don't need to customize anything, but only installing dependencies, using generators is enough for test package. Of course, if you import instead and use generate() method, is not wrong too.